### PR TITLE
Added blog

### PR DESCRIPTION
**Pull request by boceckts**
As requested at #29 

- Blogging functionality added to the jabref website with the url http://www.jabref.org/blog.
- Newest blog post gets automatically displayed in the end of the news section.
- Blog uses same design as http://www.jabref.org and http://www.jabref.org/faq/ and is generally inspired by the faq website in terms of its structure.

New blogs should be added in the _blog folder with a leading number in their name. This way they will automatically appear in the right order on the blog website (newest on the top) and the one with the highest leading number gets featured in the news section.

### CODE REVIEW

**Comment by matthiasgeiger** [comment]
Can you provide some screenshots? 

Thanks!

**Comment by koppor** [comment]
Screenshots:

grabbed_20160330-191607

and

grabbed_20160330-191658

File: _blog/005_welcome.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+title: JabRef Blog
+id: welcome
+bg: jabref-font
+color: white
+style: left
+---
+
+#Welcome to the JabRef blog!
+
+This is the blog website of JabRef. Any developer or user can write and publish blog posts on this website. To do so you need to go to <https://github.com/JabRef/www.jabref.org> and fork the repository. Then you create a new branch on your fork with the name of the subject you want to write about. Write your blog post in a markdown file name it with a leading incremented number and copy it into the _blog folder. Add your changes to your git index, commit and push your changes. Finally open a pull request and wait for feedback. Once your changes are merged your blog post will be on top of this site and featured in the news section of the [JabRef](../#news) website.
```
Comment (koppor) [inline _blog/005_welcome.md]: You have to wrap `_blog` in backticks as `_` stands for italics.

File: blog/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <li class="p-welcome"><a href="#welcome">JabRef Blog</a></li>
+        <li class="p-{{ site.blog.first.id }}"><a href="#{{ site.blog.first.id }}">Bottom</a></li>
+    </ul></nav>
+    
+
+    {% for page in site.blog reversed%}
+      {% capture id %}{{ page.id | remove:'/' | downcase }}{% endcapture %}
+      <div id="{{id}}" class="section p-{{id}}">
+        <div class="container {{ page.style }}">
+          <p class="editparagraph">
```
Comment (koppor) [inline blog/index.html]: I would not add this - since viewers should not be directly invited to modify an existing blog entry

**Comment by koppor** [comment]
I am commenting on minor things and trying out minor things, too. Thus, I'll merge it and fix the layout directly.

**Comment by tobiasdiez** [comment]
I'm missing a link on the main page to the blog .

**Comment by koppor** [comment]
The news has it. Should we also put it on the header? Should we really display the latest blog post on the main page?

**Comment by koppor** [comment]
Maybe we should replace the news entries by blog entries?

**Comment by tobiasdiez** [comment]
Personally, I would remove the news section from the main page and only link the blog.

**Comment by koppor** [comment]
I integrated it. You have to click the blog URL for yourself: The links on the top of the site are not working anymore. I'm trying to fix the next days.

**Pull request merged**